rio_tiler_pds/landsat/aws/landsat8.py

```diff
@@ -16,11 +16,10 @@
 from rio_tiler.models import ImageData
 from rio_tiler.tasks import multi_arrays
 from rio_tiler.utils import pansharpening_brovey
+from rio_tiler_pds.landsat.utils import dn_to_toa, sceneid_parser
+from rio_tiler_pds.utils import get_object
```

Comment:
Especially if you're always importing `from rio_tiler_pds.utils import get_object` I'd remove it from the top-level exports.

Reply:
this is to keep compatibility! not ideal but avoid breaking changes

Reply:
True but it was never documented as a public export, so I think it would be ok to break it. Especially because until now it was the _only_ top-level export, and all docs used fully-qualified imports.

Reply:
👌